components/list/test/list-item-placement-marker.axe.js

```diff
@@ -0,0 +1,13 @@
+import '../list-item-placement-marker.js';
+import { expect, fixture, html } from '@open-wc/testing';
+
+describe('d2l-list-item-placement-marker', () => {
+	it('visible', async() => {
+		const element = await fixture(html`<d2l-list-item-placement-marker></d2l-list-item-placement-marker>`);
+		await expect(element).to.be.accessible();
+	});
+	it('visible-rtl', async() => {
```

Comment:
I don't think we really need to check a11y for the RTL version.

Reply:
I agree, I don't think this offers any differences for accessibility from testing ltr